package.json

```diff
@@ -20,21 +20,23 @@
   "license": "MIT",
   "dependencies": {
     "alexa-app": "^3.0.0",
-    "alexa-verifier-middleware": "^0.1.8",
+    "alexa-verifier-middleware": "^0.1.9",
```

Comment:
Actually, since alexa-app already uses `alexa-verifier-middleware` already, it will probably be removed.